src/views/ColorPicker/ColorPicker.jsx

```diff
@@ -1,46 +1,24 @@
-import { useEffect, useState } from 'react';
 import Display from '../../components/Display/Display';
 import styles from './ColorPicker.css';
+import useAffirmations from '../../hooks/useAffirmation';
+import useColorPicker from '../../hooks/useColorPicker';
+
 
 export default function ColorPicker() {
-  const [fgColor, setFgColor] = useState('#ffcc00');
-  const [bgColor, setBgColor] = useState('#212121');
-  const [content, setContent] = useState('Hello, world!');
-  const [didChangeColor, setDidChangeColor] = useState(false);
-  const [affirmation, setAffirmation] = useState('');
+  //putting affirmation first so that the useColorPicker knows what bgcolor and fgcolor is from useAffirm
+  const affirmation = useAffirmations(bgColor, fgColor);
+//all of our exports from useColorPicker
+//order from the hook needs to match when we destructor it
+const [fgColor, bgColor, content, didChangeColor, handleChange] = useColorPicker();
```

Comment:
this line should be above line 9 because the variables that you are _defining_ here are being _used_ there